src/nvim/api/private/helpers.c

```diff
@@ -2015,6 +2015,16 @@ bool parse_float_config(Dictionary config, FloatConfig *fconfig, bool reconf,
         api_set_error(err, kErrorTypeValidation,
                       "Invalid value of 'style' key");
       }
+    } else if (strequal(key, "noautocmd") && new_win) {
+      if (val.type == kObjectTypeInteger) {
```

Comment:
can use `api_object_to_bool`

Reply:
Would you like me to update the other Boolean keys to use that in another commit?

Reply:
yes if you want to :)